theseus/src/api/process.rs

```diff
@@ -0,0 +1,145 @@
+//! Theseus process management interface
+use std::path::Path;
+
+use crate::state::MinecraftChild;
+pub use crate::{
+    state::{
+        Hooks, JavaSettings, MemorySettings, Profile, Settings, WindowSize,
+    },
+    State,
+};
+
```

Comment:
One thing that is missing is getting all running profiles. The main use case I see for this is displaying the running profile(s) in the app bar